### PR TITLE
Support dashboards

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Context

Tidies up some of the different dashboard names in the Performance section of Support.

## Changes proposed in this pull request

* Replace ‘Course stats’ with ‘Course statistics’
* Replace ‘UCAS matches stats’ with ‘UCAS matches’
* Changes `/support/performance/ucas-matches-stats` to `/support/performance/ucas-matches` 
* Replace ‘Reasons for rejection’ with ‘Structured reasons for rejection’
* Move feature metrics into performance namespace (also fixes main navigation item not being correctly highlighted)
* Move service performance dashboard into performance namespace
* Use `dashboard` keyword in views that use dashboard design/layout

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/docs/environment-variables.md#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/controllers/support_interface/performance_controller.rb [hunk 1]
```diff
@@ -10,6 +10,12 @@ def course_options
       @course_options = CourseOption.where('vacancy_status != ?', 'vacancies').includes(:course, :site)
     end
 
+    def dashboard
+      raise ArgumentError unless params[:year].in?([nil, '2020', '2021'])
```
Comment (duncanjbrown) [inline app/controllers/support_interface/performance_controller.rb:14]: While we're here, this seems a bit overdramatic! Maybe we should ignore non-year values not in that list? sth like

```ruby
year = params[:year] if ['2020', '2021'].include(params[:year])

@statistics = PerformanceStatistics.new(year)
```

File: config/routes.rb [hunk 2]
```diff
@@ -816,23 +816,22 @@
 
     scope '/performance' do
       get '/' => 'performance#index', as: :performance
-      get '/dashboard' => 'performance_dashboard#dashboard', as: :performance_dashboard
 
-      get '/course-stats', to: 'performance#course_stats', as: :course_stats
+      get '/dashboard' => 'performance#performance_dashboard', as: :performance_dashboard
```
Comment (duncanjbrown) [inline config/routes.rb:820]: A dashboard among dashboards — could this be `service-performance` and `service_performance_dashboard` maybe?

**Review by duncanjbrown — APPROVED** [review]
👌 Tidy and consistent — thanks @paulrobertlloyd! Minor comments inline

File: app/controllers/support_interface/performance_controller.rb (hunk 1 above)
Comment (paulrobertlloyd) [reply app/controllers/support_interface/performance_controller.rb]: Done! Seemed like it actually need to be this:

```ruby
year = params[:year] if ['2020', '2021'].include?(params[:year])
```

(`include?`)

File: config/routes.rb (hunk 2 above)
Comment (paulrobertlloyd) [reply config/routes.rb]: How does `/support/performance/service` grab you?

Comment (duncanjbrown) [reply config/routes.rb]: 🙆‍♂️ !

File: app/controllers/support_interface/performance_controller.rb [hunk 3]
```diff
@@ -10,10 +10,10 @@ def course_options
 
     def courses_dashboard; end
 
-    def performance_dashboard
-      raise ArgumentError unless params[:year].in?([nil, '2020', '2021'])
+    def service_performance_dashboard
+      year = params[:year] if ['2020', '2021'].include?(params[:year])
```
Comment (AgaDufrat) [inline app/controllers/support_interface/performance_controller.rb:14]:
```suggestion
      year = params[:year] if %w[2020 2021].include?(params[:year])
```
I think rubocop may complain about that

Comment (paulrobertlloyd) [reply app/controllers/support_interface/performance_controller.rb]: It did! Fixed 😀

**Pull request merged**